### PR TITLE
*: bump default Go version to 1.6.2

### PR DESCRIPTION
**Pull request by fabxc**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ sudo: false
 language: go
 
 go:
-- 1.5
+- 1.6.2
```
Comment (sdurrheimer) [inline .travis.yml]: Wouldn't be better to continue to test against go `1.5` and `1.6` ? People might continue to build it by hand on `go 1.5`.

```
go:
- 1.5.4
- 1.6.2
```

Comment (fabxc) [inline .travis.yml]: Ah, if that's possible. We can do that for a while.

Comment (fabxc) [inline .travis.yml]: Done

**Comment by sdurrheimer** [comment]
Is it normal the following files are always reformated by the make command:
- `template/internal/deftmpl/bindata.go`
- `ui/bindata.go`

**Comment by fabxc** [comment]
Yea... timestamps always change.

**Pull request merged**